tensorflow/python/ops/embedding_ops.py

```diff
@@ -322,6 +322,40 @@ def embedding_lookup(
   Raises:
     ValueError: If `params` is empty.
   """
+  
+  
+  """
```

Comment:
"Thank you for your feedback. You are correct, my intention was to provide additional context within the source file itself rather than having it rendered in the public documentation.